src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -113,13 +113,18 @@
 
 # candidates for enumeration
 # consts for AgentPool
-node_priorities = [CONST_SCALE_SET_PRIORITY_REGULAR, CONST_SCALE_SET_PRIORITY_SPOT]
-node_eviction_policies = [CONST_SPOT_EVICTION_POLICY_DELETE, CONST_SPOT_EVICTION_POLICY_DEALLOCATE]
+node_priorities = [CONST_SCALE_SET_PRIORITY_REGULAR,
```

Comment:
I'd appreciate it if you could restore the format of these irrelevant changes. I am refactoring the code, and this might lead to some conflicts.